test/unix/common.ml

```diff
@@ -67,26 +65,44 @@ end
 
 module Index = Index_unix.Make (Key) (Value)
 
-let log_size = 4
-
-let page_size = 2
-
-let pool_size = 2
-
-(* [tbl] is the in-memory representation of the [t], i.e. tbl should always
-contain the same elements as [t] *)
-let tbl index_name =
-  let t = Index.v ~fresh:true ~log_size index_name in
-  let tbl = Hashtbl.create 0 in
-  let rec loop i =
-    if i = 0 then (
-      Index.flush t;
-      tbl )
-    else
+module Make_context (Config : sig
+  val root : string
+end) =
+struct
+  let c = ref 0
+
+  let name_of_count () =
+    Filename.concat Config.root ("index_" ^ string_of_int !c)
```

Comment:
Maybe this would be nicer with something like this:
```ocaml
let new_name =
  let c = ref 0 in 
  fun () ->
    let n = Filename.concat Config.root ("index_" ^ string_of_int !c) in
    incr c;
    n
```

Reply:
Agreed. I've also pushed the log entry for the new name into this function too. Let me know what you think :slightly_smiling_face:

Reply:
LGTM :+1: